src/components/GameView/PointCounter.vue

```diff
@@ -0,0 +1,214 @@
+        <v-menu :location="isPlayer ? 'top' : 'bottom'">
+          <template #activator="{ props }">
+            <v-btn class="point-counter__btn" icon v-bind="props" variant="plain">
+              <v-icon
+                color="black"
+                icon="mdi-information"
+                :size="[$vuetify.display.smAndDown ? 'small' : 'large']" />
+            </v-btn>
+          </template>
+          <v-list class="score-goal-explanation">
+            <v-list-item
+              v-for="(points, idx) in kingPointsToWin"
+              :class="{ 'current-goal': kingCount === idx }"
+            >
+              {{ idx }} Kings: {{ points }}pts
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+export default {
+  name: 'PointCounter',
+  props: {
+    kingCount: {
+      required: true,
+      type: Number, // 0-4
+    },
+    pointsToWin: {
+      required: true,
+      type: Number,
+    },
+    isPlayer: {
+      default: true,
+      type: Boolean,
+    },
+  },
+  computed: {
+    ...mapGetters(['playerPointTotal', 'opponentPointTotal']),
+    currentPoints() {
+      return this.isPlayer ? this.playerPointTotal : this.opponentPointTotal;
+    },
+    kingPointsToWin() {
+      return [0, 5, 10, 14, 21];
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+
+* {
+  --main-border__color: #FFFFFF;
+  --main-text__color: rgba(27, 27, 27);
+  font-family: 'Cormorant Infant', Century Gothic, CenturyGothic, AppleGothic, sans-serif;
+}
+
+.point-counter {
+  filter: drop-shadow(2px 6px 3px rgba(50, 50, 0, 0.5));
+  line-height: 1.1;
+
+  &__wrapper {
+    position: relative;
+    background-color: rgba(255, 255, 255, 0.25);
+    width: 25px;
+    height: 100px;
+    box-sizing: border-box;
+  }
+
+  &__inner-container {
+    position: absolute;
+    font-weight: bold;
+    background-color: rgba(241, 200, 160, 0.65);
+    top: 1px;
+    left: 1px;
+    width: 23px;
+    height: 98px;
+    color: rgba(27, 27, 27);
+  }
+
+  &__btn {
+    margin-top: -0.8em;
+  }
+
+  &__current-points {
+    font-size: 22px;
+    border-bottom: 1px solid rgba(27, 27, 27);
+  }
+
+  &__total-points {
+    font-size: 16px;
+  }
+
+  &__player {
+    clip-path: polygon(0 0, 100% 0, 100% 90%, 50% 95%, 0 90%);
+  }
+
+  &__opponent {
+    clip-path: polygon(0% 100%, 100% 100%, 100% 10%, 50% 5%, 0 10%);
+  }
+
+  &__current-opponent {
+    margin-top: 12px;
+  }
+
+  &__current-player {
+    margin-top: 9px;
+  }
+
+  &__current-opponent {
+    margin-top: 16px;
+  }
+}
+
+.current-goal {
+  background-color: rgb(115, 160, 110);
+}
+@media only screen and (max-device-width: 400px) and (orientation: portrait) {
+  .point-counter {
+    line-height: 1.1;
+
+    &__wrapper {
+      width: 30px;
+      height: 125px;
+    }
+
+    &__inner-container {
+      top: 2px;
+      left: 2px;
+      width: 26px;
+      height: 121px;
+    }
```

Comment:
Same for these, the values are all over the place. I know why you're doing it, to get the visuals "just right", but it will make this a bear to update and maintain in the future. I don't think there's a particular fix for it right now without deep diving on it, and I don't think it's a blocker, but it is something we should align on longterm.

We probably need a longer term design overhaul imo, at least in terms of standards. Some sort of a Cuttle design system with logical defaults and global values.